### PR TITLE
Updated about_Class.HostManagement

### PR DESCRIPTION
**Pull request by guptaravi540**
Corrected some spellings, punctuation marks and blank spaces.

### CODE REVIEW

File: about_Class.HostsManagement.help.txt [hunk 1]
```diff
@@ -20,29 +20,28 @@ LONG DESCRIPTION
...
       Represents the type of the HOSTS Entry line.
 
-
       Structure:
 
       [HostsFile]
         |--> [HostsEntry]
         |--> [HostsEntry]
         |--> [HostsEntry]
-                    |--> [HostsentryType]
```
Comment (Stephanevg) [inline about_Class.HostsManagement.help.txt]: The TAB here was intentional, as it describes an indentation. A deeper level.
Please remove this.

Comment (guptaravi540) [reply about_Class.HostsManagement.help.txt]: Okay Sure.

**Review by guptaravi540 — COMMENTED** [review]
Please merge it.

**Comment by Stephanevg** [comment]
Hi, @guptaravi540 I am sorry, but I ahve merged some other commits a bit earlier today. which result in a conflict now. 

AS these are some minor changes, would you mind pulling the data back, and doing another PR whit your changes, that would help me very much.

Thanks!

**Pull request closed without merging**